samples/glean/src/main/java/org/mozilla/samples/glean/MainActivity.kt

```diff
@@ -32,5 +38,10 @@ open class MainActivity : AppCompatActivity() {
                     )
             )
         }
+
+        // Generate a "baseline" ping on click.
+        findViewById<View>(R.id.buttonSendPing).setOnClickListener {
+            Glean.handleEvent(Glean.PingEvent.Background)
+        }
```

Comment:
nit: Kotlin Android Extensions are enabled for this sample app. You should be able to just use `buttonSendPing` and `buttonWebView`. Android Studio should then offer to import that from `kotlinx.android.synthetic.main.activity_main.`. With that no `findViewById` is needed. :)

Reply:
Ah, nice! AndroidStudio didn't offer me that, but everything works fine after manually importing it!